packages/amplication-server/src/core/resource/resource.service.ts

```diff
@@ -580,6 +599,30 @@ export class ResourceService {
       const newResourcesMap = new Map<string, Resource>();
 
       for (const newService of newServices) {
+        const adminPathWithoutLastFolder = originalResourceAdminPath?.substring(
+          0,
+          originalResourceAdminPath.lastIndexOf("/") + 1
+        );
+
+        const serverPathWithoutLastFolder =
+          originalResourceServerPath?.substring(
+            0,
+            originalResourceServerPath.lastIndexOf("/") + 1
+          );
+
+        const baseAdminPath =
+          !originalResourceId || !originalResourceAdminPath
+            ? ""
+            : !originalResourceAdminPath.includes("/")
+            ? `${newService.name}-admin`
+            : `${adminPathWithoutLastFolder}${newService.name}-admin`;
+        const baseAServerPath =
+          !originalResourceId || !originalResourceAdminPath
+            ? ""
+            : !originalResourceServerPath.includes("/")
```

Comment:
originalResourceServerPath might be undefined here?

Reply:
@mulygottlieb  no, because of the first check:  !originalResourceId || !originalResourceAdminPath

Reply:
but I see that I need to update the  to originalResourceServerPath